rdrf/registry/patients/models.py

```diff
@@ -1118,6 +1125,15 @@ def family(self):
         return Family(self)
 
 
+class PatientStage(models.Model):
+    name = models.CharField(max_length=200)
+    allowed_prev_stages = models.ManyToManyField('self', symmetrical=False, related_name='prev_stages')
+    allowed_next_stages = models.ManyToManyField('self', symmetrical=False, related_name='next_stages')
```

Comment:
`related_name` is the inverse relationship so they should be called accordingly.

Reply:
Updated.